frontends/common/resolveReferences/resolveReferences.h

```diff
@@ -17,9 +17,9 @@ limitations under the License.
 #ifndef COMMON_RESOLVEREFERENCES_RESOLVEREFERENCES_H_
 #define COMMON_RESOLVEREFERENCES_RESOLVEREFERENCES_H_
 
+#include "absl/container/flat_hash_map.h"
```

Comment:
Actually, can we make absl a bracketed system include to be consistent?

Reply:
I think someone asked somewhere exactly the opposite for fetched dependencies :)

Reply:
Need to find. But I'm happy to change to angled brackets. This makes much more sense to me :)

Reply:
Ok, apparently abseil expects to use quoted includes. And bazel build even enforces this (the path is exported as `-iquoted`).

Reply:
Ugh, seems to be an opinionated maintainer. This might cause issues with Werror and clang-tidy. Not a big deal but annoying.

We can just patch this, I can get this done in a separate PR.